crates/openssl-src-111/src/openssl.rs

```diff
@@ -134,6 +147,12 @@ impl Build {
 
         clone_repo(&inner_dir.to_str().unwrap()).unwrap();
 
+        if cfg!(feature = "openssl312") {
+            let source_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+            // Patching the repo to get deterministic timestamp
+            patch_openssl(out_dir, "det_timestamp.patch").unwrap();
+        }
```

Comment:
This seems kind of dangerous to enable by default

Reply:
Will be removed. That was an attempt at making OpenSSL fully deterministic, even the timestamps. DId not work in the end.